swagger-spec/users/external_identities_detail.yaml

```diff
@@ -0,0 +1,96 @@
+get:
+  summary: Retrieve user's external identities
+  description: >-
+    Retrieves a JSON object of a user's external identity, currently this only includes their ORCID account.
+
+    The returned information includes the external identity's name, status and id.
+
+    #### Returns
+
+    Returns a JSON object with a `data` key containing the representation of the requested
+    external identities, if the request is successful.
+
+    If the request is unsuccessful, an `errors` key containing
+    information about the failure will be returned. Refer to the [list of error codes](#tag/Errors-and-Error-Codes)
+    to understand why this request may have failed.
+
+
+  parameters:
+  - in: path
+    type: string
+    name: user_id
+    required: true
+    description: 'The unique identifier of the user.'
+  - in: path
+    type: string
+    name: identities_id
+    required: true
+    description: "The unique identifier of a user's external identity."
+  tags:
+    - Users
+  operationId: external_identities_detail
+  responses:
+    '200':
+      description: 'OK'
+      schema:
+        $ref: definition.yaml
+      examples:
+        application/json:
+          data:
+            links:
+              self: https://api.osf.io/v2/users/q7fts/settings/identities/ORCID/
+            attributes:
+              status: VERFIED
+              external_id: '0000-0001-9143-4652'
+              id: ORCID
+            type: users
+            id: q7fts
+
+delete:
+  summary: Delete user's external identities
+  description: >-
+    Deletes  a user's external identity.
+
+    The returns deleted information including the external identity's name, status and id.
+
+    #### Returns
```

Comment:
See other delete request sections, I think this should be:
```
#### Returns

If the request is successful, no content is returned.


If the request is unsuccessful, a JSON object with an errors key containing information about the failure will be returned. Refer to the [list of error codes](#tag/Errors-and-Error-Codes) to understand why this request may have failed.
```

Reply:
Like other sections, please add a "Permissions" section to explain that users can only delete their own identities.